src/library.cpp

```diff
@@ -3159,6 +3159,114 @@ void lammps_gather_bonds(void *handle, void *data)
   END_CAPTURE
 }
 
+// Gather function for angles
+void lammps_gather_angles(void *handle, void *data)
```

Comment:
There need to be doxygen comments added here to document the function. Same for the others below.

Reply:
I think I have added the required doxygen comments in the library.cpp